src/styles/utils/general.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  margin: 0;
```

Comment:
no need to set font-family  and margin for all selectors

Reply:
Ok, sir! :D